src/logic/translations.js

```diff
@@ -227,6 +228,37 @@ const texts = {
       '333mbf': 'Puzieurs 3x3x3 bôgné',
     },
   },
+  pl: {
+    eventLabel: 'Konkurencja',
+    round: 'Runda',
+    group: 'Grupa',
+    station: 'Stanowisko',
```

Comment:
This is going to overflow (enabled in `Configure > General > Print out station number`)

```suggestion
    station: 'Stan',
```